addons/rearm/stringtable.xml

```diff
@@ -612,5 +612,13 @@
             <Italian>Il rifornimento delle munizioni (-1 per disabilitarlo)</Italian>
             <Russian>Объем боеприпасов для перевооружения (-1 для отмены)</Russian>
         </Key>
+        <Key ID="STR_ACE_Rearm_RearmSettings_distance_DisplayName">
+            <English>Rearm distance</English>
+            <French>Distance de réarmement</French>
```

Comment:
```suggestion
            <French>Distance de réarmement</French>
            <German>Aufmunitionierungs Reichweite</German>
```
Might be too long?
Might also do `Aufmunitionierungsreichweite` but pls no.

Reply:
definitely too long

Reply:
Maybe @commy2 has an idea

Reply:
Aufmunitionierungsreichweite (28)
Nachschubsreichweite (20)
Nachschubsbereich (17)
Aufrüstungsbereich (17)

some ideas...

Reply:
first one could clip, the other three would work, just give me the best one and i'll add it.

Reply:
Nachschubbereich would be without s, as schubs is pushing someone, would mean after push area.
Personally I'd just keep Rearm probably :D 

```suggestion
            <French>Distance de réarmement</French>
            <German>Aufrüstbereich</German>
```